src/components/newsFeedV2/newsFeed.css

```diff
@@ -0,0 +1,91 @@
+@import '../app/variables.css';
```

Comment:
IMO we shouldn't use anything from the old variables in the new design component.